src/util/run-plugin-promise.js

```diff
@@ -21,16 +21,20 @@ module.exports = function runPluginFunction(params, functionName) {
         } else {
           pluginName = plugin
         }
-
-        var tmp = require(pluginName)
-        var has = tmp.hasOwnProperty(functionName)
-        var fn = tmp[functionName]
+        if (pluginName.split('').includes('/')) {
+          pluginName = '@' + pluginName
+        }
+        // Must require from full path from via current Architect process
+        var req = process.cwd() + '/node_modules/' + pluginName
```

Comment:
I think `path.join(...)` would be better to support multiple platforms (e.g. Windows)

Reply:
Hmm, excellent catch, require additional interpolation in there!